deploy/1_assets_bucket.yaml

```diff
@@ -0,0 +1,61 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: TTUI assets S3 bucket
+Outputs:
+  AssetsBucket:
+    Export:
+      Name:
+        Fn::Sub: ttui-assets-AssetsBucket
+    Value:
+      Ref: AssetsBucket
+  AssetsBucketDomainName:
+    Export:
+      Name:
+        Fn::Sub: ttui-assets-AssetsBucketDomainName
+    Value:
+      Fn::GetAtt:
+        - AssetsBucket
+        - DomainName
+  SyncUserAccessKeyId:
+    Export:
+      Name:
+        Fn::Sub: ttui-assets-SyncUserAccessKeyId
+    Value:
+      Ref: SyncUserAccessKey
+  SyncUserAccessKeySecret:
+    Export:
+      Name:
+        Fn::Sub: ttui-assets-SyncUserAccessKeySecret
+    Value:
+      Fn::GetAtt:
+        - SyncUserAccessKey
+        - SecretAccessKey
+Resources:
+  AssetsBucket:
+    Properties:
+      BucketName: ttui-assets
```

Comment:
This doesn't belong here.

Shouldn't this be in `design`?

Also please use Terraform for new stuff.

Reply:
I am against this because everything else web-related uses CloudFormation, but okay